src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+import { TodoFilter } from '../TodoFilter';
+import { FilterOption } from '../../enums/FilterOption';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filterOption: FilterOption;
+  setFilterOption: (filterOption: FilterOption) => void;
+  removeCompleted: () => void;
+}
+
+export const Footer: FC<Props> = ({
+  todos,
+  filterOption,
+  setFilterOption,
+  removeCompleted,
+}) => {
+  const completedTodos = todos.filter((todo) => todo.completed);
+  const activeTodos = todos.filter((todo) => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodos.length} items left`}
+      </span>
```

Comment:
You can add option for 1 item if length === 1 (1 **item** left)

Reply:
@EugeneSherepa sorry, I don't understand what do you mean?

Reply:
@vladyslav-lapshyn 
```suggestion
       <span className="todo-count">
        {todos.length === 1
          ? '1 item left'
          : `${todos.length} items left`}
      </span>
```